### PR TITLE
Regenerate Trace API client for nodejs.

### PR DESCRIPTION
**Pull request by jmuk**

### CODE REVIEW

**Comment by jmuk** [comment]
@shinfan FYI

File: generated/nodejs/google-cloud-node/packages/trace/package.json [hunk 1]
```diff
@@ -3,15 +3,16 @@
   "name": "@google-cloud/trace",
   "version": "0.7.1",
   "author": "Google Inc.",
-  "description": "Stackdriver Trace library for Node.js",
+  "description": "Trace library for Node.js",
```
Comment (landrito) [inline generated/nodejs/google-cloud-node/packages/trace/package.json]: Similarly, this should be `Stackdriver`.

File: generated/nodejs/google-cloud-node/packages/trace/README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
-Stackdriver Trace API for Node.js
+Google Trace API for Node.js
```
Comment (jmuk) [inline generated/nodejs/google-cloud-node/packages/trace/README.md]: Fixed

File: generated/nodejs/google-cloud-node/packages/trace/package.json (hunk 1 above)
Comment (jmuk) [inline generated/nodejs/google-cloud-node/packages/trace/package.json]: Fixed

**Pull request merged**